### ISSUE TITLE
prop-types rule should not trigger error for attributes

### ISSUE THREAD

**Issue by jbe456** [yannickcr/eslint-plugin-react#135]
Using version `2.6.2` and the rule `react/prop-types`:

```
"react/prop-types": [2, {"ignore": ["foo"]}],
```

Then I have error if in my code I'm doing:
```
this.props.foo.bar
```
```
'foo.bar' is missing in props validation for XYZ
```

Although I would have expected not to trigger any error since I asked to ignore foo.
